apps/admin/backend/test/set_env_vars.ts

```diff
@@ -0,0 +1,3 @@
+process.env['VX_MACHINE_TYPE'] = 'admin';
```

Comment:
Artifact authentication relies on this variable, which is guaranteed to be set on all real machines

Reply:
Maybe put this comment in this file for posterity?

Reply:
Good call: https://github.com/votingworks/vxsuite/pull/3859/commits/81f1edb8d55ec0a507d6af9f5dc8140663846999